### DISCUSSION TITLE
Showing incorrect commit author for GitHub App

### DISCUSSION THREAD

**Discussion by jackycute** [community/community#30492] [Other Feature Feedback, Questions, & Ideas]
I found an issue that the commit made by our GitHub App is showing as another user we never know.
Here is the link and screenshot: https://github.com/hackmdio/docs-with-hackmd/commit/a689e5ad0f8a1b5e19f125e4d868692789ab21aa

![image](https://user-images.githubusercontent.com/2639151/186081035-180349aa-ecf3-4a69-bca7-1c720f83f4a8.png)

That user [kjmoutlook](https://github.com/hackmdio/docs-with-hackmd/commits?author=kjmoutlook) joined GitHub only a month ago but the commit was made 3 years ago.
Dose anyone have same issue? It brothers me and our users a lot.

**Comment by peter-evans** [comment]
That user has probably registered with GitHub using the same email address that was used to make the commits. I think you can have this resolved by contacting GitHub support.

See this article which explains how this happens.
https://repography.com/blog/how-to-take-credit-for-someone-elses-work-on-github

**Reply by jackycute** [reply]
Thank you @peter-evans.
I have opened a ticket to GitHub Support but not receiving any reply for 4 days already, will try to push again.
